README.md

```diff
@@ -6,15 +6,27 @@ Installing Webpack and TypeScript globally
 
     $ npm install -g webpack
     $ npm install -g typescript
+    $ npm install
 
 ### Easy development build
 - Build source scripts
 
-    `$ webpack`
+    `$ npm run dev`
 
 - Serve up the new library view
 
-    `$ node .\index.js`
+    `$ node ./index.js`
+
+- Navigate to `localhost:3456` in Google Chrome browser
+
+### Easy production build
+- Build source scripts
+
+    `$ npm run build`
+
+- Serve up the new library view
+
+    `$ node ./index.js`
```

Comment:
Let's merge both `Easy production build` and `Easy development build` sections into one `Building librarie.js`. Then under *Build source scripts* we have:

    `$ npm run dev` (for development build)
    `$ npm run prod` (for production build)

Oh yes, change that `npm run build` to `npm run prod` for clarity.